pkg/tui/table.go

```diff
@@ -165,6 +176,13 @@ func (t *Table) initTable() {
 			}
 		}).
 		SetInputCapture(func(ev *tcell.EventKey) *tcell.EventKey {
+			if ev.Rune() == 'r' {
```

Comment:
Let's use standard `CTRL + R` / `F5` hotkeys for this

```suggestion
			if ev.Key() == tcell.KeyCtrlR || ev.Key() == tcell.KeyF5 {
```

Reply:
Thanks for reviewing @ankitpokhrel ! alright I will proceed make a few more commits to address your comments above 👍🏻